webofneeds/won-owner/src/main/java/won/owner/service/impl/AuthenticationThreadLocal.java

```diff
@@ -0,0 +1,30 @@
+package won.owner.service.impl;
+
+import org.springframework.security.core.Authentication;
+
+/**
+ * Class that exposes the Authentication to the current thread.

```

Comment:
why is this info passed through the thread? i dont really understand this

Reply:
It's hacky, I agree. I have no idea how to better do it - I've tried. The threadlocal is set, then the method that needs it down the call stack is called, and then it's removed again. The use is quite limited - but I don't like it, either. I suggest we keep it like this for now, though

Reply:
i also dont know a better solution